tf2_kdl/CMakeLists.txt

```diff
@@ -35,18 +46,12 @@ if(BUILD_TESTING)
   find_package(tf2_msgs REQUIRED)
 
   ament_add_gtest(test_kdl test/test_tf2_kdl.cpp)
-  target_include_directories(test_kdl PUBLIC
-    include
-  )
-  ament_target_dependencies(test_kdl
-    builtin_interfaces
-    orocos_kdl
-    rclcpp
-    tf2
-    tf2_ros
-    tf2_msgs)
+  target_link_libraries(test_kdl
+    tf2_kdl
+    rclcpp::rclcpp
+    ${tf2_msgs_TARGETS})
```

Comment:
This test also depends on tf2, tf2_ros, and kdl.  Shouldn't we depend on them here?

Reply:
In https://github.com/ros2/geometry2/pull/493/commits/7dac8dd51997b0d0892a0e527e12c5f738b98d24 , https://github.com/ros2/geometry2/pull/493/commits/2a9df115fcde75dbd54ddc9347e55d66c1dbedd7 , and https://github.com/ros2/geometry2/pull/493/commits/e78a34d4724545ce557b9cf568d66819dfcd6c6d I double checked the dependencies, and applied the method we came up with in #496 of documenting some dependencies of tests instead of linking them.